tutorials/plot_brainstorm_auditory.py

```diff
@@ -0,0 +1,356 @@
+    - Random ISI: between 0.7s and 1.7s seconds, uniformly distributed.
+    - Button pressed when detecting a deviant with the right index finger.
+
+The specifications of this dataset were discussed initially on the FieldTrip
+bug tracker:
+http://bugzilla.fcdonders.nl/show_bug.cgi?id=2300
+
+References
+----------
+.. [1] Tadel F, Baillet S, Mosher JC, Pantazis D, Leahy RM.
+       Brainstorm: A User-Friendly Application for MEG/EEG Analysis.
+       Computational Intelligence and Neuroscience, vol. 2011, Article ID
+       879716, 13 pages, 2011. doi:10.1155/2011/879716
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import os.path as op
+import csv
+
+import mne
+from mne.datasets.brainstorm import bst_auditory
+from mne.io import read_raw_ctf
+from mne.filter import notch_filter, low_pass_filter
+
+print(__doc__)
+
+
+# To reduce memory consumption and running time, some of the steps are
+# precomputed. To run everything from scratch change this to False.
+use_precomputed = True
+
+##############################################################################
+# The data was collected with a CTF 275 system at 2400 Hz and low-pass
+# filtered at 600 Hz. Here the data and empty room data files are read to
+# construct instances of :class:`mne.io.Raw`.
+data_path = bst_auditory.data_path()
+
+subject = 'bst_auditory'
+subjects_dir = op.join(data_path, 'subjects')
+
+raw_fname1 = op.join(data_path, 'MEG', 'bst_auditory',
+                     'S01_AEF_20131218_01.ds')
+raw_fname2 = op.join(data_path, 'MEG', 'bst_auditory',
+                     'S01_AEF_20131218_02.ds')
+erm_fname = op.join(data_path, 'MEG', 'bst_auditory',
+                    'S01_Noise_20131218_01.ds')
+preload = not use_precomputed
+raw1 = read_raw_ctf(raw_fname1, preload=preload)
+raw2 = read_raw_ctf(raw_fname2, preload=preload)
+
+raw_erm = read_raw_ctf(erm_fname, preload=preload)
+
+raws = [raw1, raw2, raw_erm]
+##############################################################################
+# Data channel array consisted of 274 MEG axial gradiometers, 26 MEG reference
+# sensors and 2 EEG electrodes (Cz and Pz).
+# In addition:
+#   - 1 stim channel for marking presentation times for the stimuli
+#   - 1 audio channel for the sent signal
+#   - 1 response channel for recording the button presses
+#   - 1 ECG bipolar
+#   - 2 EOG bipolar (vertical and horizontal)
+#   - 12 head tracking channels
+#   - 20 unused channels
+# The head tracking channels and the unused channels are marked as misc
+# channels. Here we define the EOG and ECG channels.
+for raw in raws[:2]:
+    raw.set_channel_types({'HEOG': 'eog', 'VEOG': 'eog', 'ECG': 'ecg'})
+    if not use_precomputed:
+        # Leave out the two EEG channels for easier setup of source space.
```

Comment:
Setting up the source space shouldn't have any interaction with the presence of EEG channels. You mean computation of forward perhaps?